### DISCUSSION TITLE
Will filecoin support multi-addresses transfer?

### DISCUSSION THREAD

**Discussion by shaodan** [filecoin-project/lotus#4742] [Storage Provider] [answered]
Currently ```lotus send``` only  support one-to-one transaction, unlike other blockchains(e.g. bitcoin)

**Comment by jennijuju** [comment]
This is interesting,  however, there are many things to consider about and would require a FIP to add a new message type

**Comment by Kubuxu** [comment, marked as answer]
Filecoin most likely won't support one-to-many transfers.
Bitcoin uses completly different way of tracking balances which causes `many-to-many` transfers to be a necessity.
One-to-many transfers might be supported whenever user programmable actors are implemented but that is still some ways away.

**Comment by dd45e640b42e6da7da96faee3996ef7c** [comment]
@shaodan do you actually want a multi-multi transaction on the chain or do you want a client side 

`lotus send --from=addr1 --from=addr2 targetAddr 123`
`lotus send --from=addr  targetAddr1 targetAddr2 1234`
...
?

client side a `multi to one` and even `multi to multi` can be faked quite easily

@Kubuxu aren't a SubmitProveCommit and SubmitPreCommit messages `one to many` messages in the end already?
